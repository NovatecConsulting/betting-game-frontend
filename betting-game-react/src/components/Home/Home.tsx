import React from "react";

const Home: React.FC = () => (
    <div className="min-h-screen min-w-screen bg-white text-center text-6xl text-black grid items-center tracking-wider">
        Home Component
    </div>
);
export default Home;
