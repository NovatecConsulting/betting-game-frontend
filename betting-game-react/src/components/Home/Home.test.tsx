import React from "react";
import { shallow } from "enzyme";
import Home from "./Home";

describe(Home.name + " component", () => {
    it("should render and match snapshot with correct content text.", () => {
        const wrapper = shallow(<Home />);

        expect(wrapper.find("div")).toIncludeText("Home Component");
        expect(wrapper).toMatchSnapshot();
    });
});
