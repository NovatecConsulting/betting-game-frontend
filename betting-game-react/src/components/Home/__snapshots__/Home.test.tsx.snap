// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component should render and match snapshot with correct content text. 1`] = `
<div
  className="min-h-screen min-w-screen bg-white text-center text-6xl text-black grid items-center tracking-wider"
>
  Home Component
</div>
`;
